### ISSUE TITLE
Can't move Jellyfin container from 10.6.4 to 10.7-RC1 for parallel servers.

### ISSUE THREAD

**Issue by ABotelho23** [jellyfin/jellyfin#4676]
**Description**
Can't move Jellyfin container from 10.6.4 to 10.7-RC1

**System:**
 - OS: CentOS 8
 - Virtualization: Docker/Podman
 - Browser: Firefox 79.0
 - Jellyfin Version: 10.6.4 -> 10.7-RC1
 - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
 - Reverse Proxy: Caddy
 - Base URL: none
 - Storage: NFS

**To Reproduce**
1. Copy contents of Jellyfin 10.6.4 to 10.7-RC1: `cp -aR jellyfin/* jellyfinbeta/`
2. Create 10.7-RC1 container: `podman create --name=jellyfinbeta --volume /srv/jellyfinbeta/config:/config --volume /srv/jellyfinbeta/cache:/cache --volume /mnt/media:/media -p 8097:8096 jellyfin/jellyfin:stable-rc`
3. Start the 10.7-RC1 container: `podman start jellyfinbeta`
4. load up _beta1.example.com_ in web browser.
5. Error: 502 Bad Gateway, blank page.
The character encoding of the plain text document was not declared. The document will render with garbled text in some browser configurations if the document contains characters from outside the US-ASCII range. The character encoding of the file needs to be declared in the transfer protocol or file needs to use a byte order mark as an encoding signature. beta1.example.com
 
**Expected behavior**
Jellyfin login page loads.

**Additional context**
I intend to run both stable (10.6.4) and RC (10.7-RC1) side-by-side.

**Comment by crobibero** [comment]
Please post the server log, the browser log isn’t helpful in this context

**Comment by ABotelho23** [comment]
I don't get all that much. My stable container's log is is happily chugging along, but the rc one just spits out this:
**RC LOG**
`[2020-12-05 22:23:38.748 +00:00] [INF] [1] Main: Jellyfin version: "10.7.0"
[2020-12-05 22:23:38.764 +00:00] [INF] [1] Main: Environment Variables: ["[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /config]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_MEDIA_DIR, /media]", "[DOTNET_SYSTEM_GLOBALIZATION_INVARIANT, 1]"]
[2020-12-05 22:23:38.767 +00:00] [INF] [1] Main: Arguments: ["/jellyfin/jellyfin.dll"]
[2020-12-05 22:23:38.767 +00:00] [INF] [1] Main: Operating system: "Linux"
[2020-12-05 22:23:38.767 +00:00] [INF] [1] Main: Architecture: X64
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: 64-Bit Process: True
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: User Interactive: True
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: Processor count: 2
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: Program data path: "/config"
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: Web resources path: "/jellyfin/jellyfin-web"
[2020-12-05 22:23:38.768 +00:00] [INF] [1] Main: Application directory: "/jellyfin/"
[2020-12-05 22:23:38.923 +00:00] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "/cache"
[2020-12-05 22:23:38.954 +00:00] [ERR] [1] Jellyfin.Networking.Manager.NetworkManager: No interfaces information available. Resolving DNS name.
[2020-12-05 22:23:38.968 +00:00] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses : "[10.23.0.0/16,172.23.0.0/16,127.0.0.1/32]"
[2020-12-05 22:23:38.968 +00:00] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions : "[]"
[2020-12-05 22:23:38.969 +00:00] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: "[10.23.0.0/16,172.23.0.0/16,127.0.0.1/32]"
[2020-12-05 22:23:38.970 +00:00] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: "[]"
[2020-12-05 22:23:38.970 +00:00] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind exclusions: "[]"
[2020-12-05 22:23:38.986 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[2020-12-05 22:23:39.042 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "LDAP-Auth, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/LDAP Authentication/LDAP-Auth.dll"
[2020-12-05 22:23:39.043 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "Novell.Directory.Ldap.NETStandard, Version=3.2.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/LDAP Authentication/Novell.Directory.Ldap.NETStandard.dll"
[2020-12-05 22:23:39.046 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "OpenSubtitlesHandler, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/Open Subtitles/OpenSubtitlesHandler.dll"
[2020-12-05 22:23:39.047 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "Jellyfin.Plugin.OpenSubtitles, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/Open Subtitles/Jellyfin.Plugin.OpenSubtitles.dll"`

**STABLE LOG**
`[2020-12-05 22:26:38.025 +00:00] [INF] [1] Main: Jellyfin version: "10.6.4"
[2020-12-05 22:26:38.041 +00:00] [INF] [1] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_MEDIA_DIR, /media]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /config]", "[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[DOTNET_SYSTEM_GLOBALIZATION_INVARIANT, 1]"]
[2020-12-05 22:26:38.045 +00:00] [INF] [1] Main: Arguments: ["/jellyfin/jellyfin.dll"]
[2020-12-05 22:26:38.045 +00:00] [INF] [1] Main: Operating system: "Linux"
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: Architecture: X64
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: 64-Bit Process: True
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: User Interactive: True
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: Processor count: 2
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: Program data path: "/config"
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: Web resources path: "/jellyfin/jellyfin-web"
[2020-12-05 22:26:38.046 +00:00] [INF] [1] Main: Application directory: "/jellyfin/"
[2020-12-05 22:26:38.184 +00:00] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "/cache"
[2020-12-05 22:26:38.197 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[2020-12-05 22:26:38.206 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "OpenSubtitlesHandler, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/Open Subtitles/OpenSubtitlesHandler.dll"
[2020-12-05 22:26:38.208 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "Jellyfin.Plugin.OpenSubtitles, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/Open Subtitles/Jellyfin.Plugin.OpenSubtitles.dll"
[2020-12-05 22:26:38.208 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "LDAP-Auth, Version=9.0.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/LDAP Authentication/LDAP-Auth.dll"
[2020-12-05 22:26:38.209 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loaded assembly "Novell.Directory.Ldap.NETStandard, Version=3.2.0.0, Culture=neutral, PublicKeyToken=null" from "/config/plugins/LDAP Authentication/Novell.Directory.Ldap.NETStandard.dll"
[2020-12-05 22:26:39.647 +00:00] [INF] [1] Main: Kestrel listening on all interfaces
[2020-12-05 22:26:39.745 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Running startup tasks
[2020-12-05 22:26:39.784 +00:00] [INF] [1] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for "Extract Chapter Images" set to fire at 12/06/2020 02:00, which is 3:33:20.215826 from now.
[2020-12-05 22:26:39.821 +00:00] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version "4.3.1"
[2020-12-05 22:26:39.845 +00:00] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available "decoders": ["h264", "h264_qsv", "h264_cuvid", "hevc", "hevc_qsv", "hevc_cuvid", "mpeg2video", "mpeg2_qsv", "mpeg2_cuvid", "mpeg4", "mpeg4_cuvid", "msmpeg4", "vc1_qsv", "vc1_cuvid", "vp8_cuvid", "vp8_qsv", "vp9_cuvid", "vp9_qsv", "aac", "ac3", "mp3"]
[2020-12-05 22:26:39.861 +00:00] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available "encoders": ["libx264", "h264_amf", "h264_nvenc", "h264_qsv", "h264_v4l2m2m", "h264_vaapi", "libx265", "hevc_amf", "hevc_nvenc", "hevc_qsv", "hevc_vaapi", "mpeg4", "msmpeg4", "aac", "ac3", "libmp3lame", "libopus", "libvorbis", "srt"]
[2020-12-05 22:26:39.876 +00:00] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["vdpau", "cuda", "vaapi", "qsv", "drm", "opencl"]
[2020-12-05 22:26:39.876 +00:00] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: Custom: "/usr/lib/jellyfin-ffmpeg/ffmpeg"
[2020-12-05 22:26:39.878 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: ServerId: "71ab0ac8d6a84cb88949c1a3c28fa7f9"
[2020-12-05 22:26:39.972 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all pre-startup entry points in 0:00:00.084863
[2020-12-05 22:26:39.972 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Core startup complete
[2020-12-05 22:26:40.131 +00:00] [INF] [10] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /media/Videos/Movies
[2020-12-05 22:26:40.138 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all post-startup entry points in 0:00:00.1647127
[2020-12-05 22:26:40.138 +00:00] [INF] [1] Main: Startup complete 0:00:02.2597618
[2020-12-05 22:26:40.160 +00:00] [INF] [6] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /media/Videos/TV Series
[2020-12-05 22:26:42.787 +00:00] [INF] [6] Emby.Server.Implementations.ScheduledTasks.TaskManager: "StartupTrigger" fired for task: "Update Plugins"
[2020-12-05 22:26:42.788 +00:00] [INF] [6] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queueing task "PluginUpdateTask"
[2020-12-05 22:26:42.791 +00:00] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Update Plugins"
[2020-12-05 22:26:43.569 +00:00] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: "Update Plugins" Completed after 0 minute(s) and 0 seconds
[2020-12-05 22:26:44.007 +00:00] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[2020-12-05 22:28:01.202 +00:00] [INF] [4] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:127.0.0.1" request
[2020-12-05 22:28:04.641 +00:00] [INF] [4] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:127.0.0.1" closed
[2020-12-05 22:28:05.793 +00:00] [INF] [4] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS "::ffff:127.0.0.1" request
[2020-12-05 22:28:06.491 +00:00] [ERR] [17] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: "A task was canceled". URL: "http://media.example.com/Items/d6e74c8f8f077d688de4fabeb60a090e/Images/Primary?maxHeight=891&maxWidth=594&tag=0e372ee656fe0421caf5aaafdf1244c2&quality=90"
[2020-12-05 22:28:06.491 +00:00] [ERR] [16] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: "A task was canceled". URL: "http://media.example.com/Items/a7396d16c9f138361e20fae3aa264ece/Images/Primary?maxHeight=891&maxWidth=594&tag=b5eb9ef7da3b7f8adb6a03d145928f28&quality=90"
[2020-12-05 22:28:06.492 +00:00] [ERR] [4] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: "A task was canceled". URL: "http://media.example.com/Items/c52e07556ad11de5f5b8bf73292d539b/Images/Primary?maxHeight=891&maxWidth=594&tag=2d604179f6f975ed853863c186c663c7&quality=90"
`

**Comment by ABotelho23** [comment]
Something curious. I removed my installed plugins (LDAP and OpenSubtitles) from my stable installation, and re-performed the copy over. I was able to fire up the 10.7-RC1 build!
I went back to the stable build, and noticed the LDAP plugin was locked to 9.0.0.0 for 10.6.4. Trying to install LDAP 10.0.0.0 would say "package not found" in the server log. But, on the 10.7-RC1 build, I was able to install the LDAP 10.0.0.0 plugin. So this kind of sounds like a migration issue. Are plugin migrations not a thing? Are we meant to remove all plugins before a major update and reinstall them after? Would this not be best to display an error about a failure in plugins and correctly initialize the server anyway?
